src/app/page.tsx

```diff
@@ -5,6 +5,7 @@ import {
 	HowItWorksSection,
 	RoadmapSection,
 } from "@/components/04-templates";
+import { TheFooterSection } from "@/components/04-templates/TheFooterSection";
```

Comment:
hey yo @Hardsys-Andre ! 
Maybe we can import TheFooterSection like the others
``@/components/04-templates"``